### PR TITLE
Defines $262.agent.monotonicNow()

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

**Comment by rwaldron** [comment]
Implementations: 


- [x] [JSC](https://bugs.webkit.org/show_bug.cgi?id=185043)
- [ ] [SpiderMonkey](https://bugzilla.mozilla.org/show_bug.cgi?id=1457560)
- [x] [V8](https://chromium-review.googlesource.com/c/v8/v8/+/1033426)

File: INTERPRETING.md [hunk 1]
```diff
@@ -101,6 +102,16 @@ properties of the global scope prior to test execution.
...
         sleeps the execution for approximately that duration.
+    - **`monotonicNow`** - a function that returns a value that conforms to `DOMHighResTimeStamp` and is produced in such a way that its semantics conform to **Monotonic Clock**. 
+
+
+#### Normative references
+
+[**DOMHighResTimeStamp**](https://www.w3.org/TR/hr-time-2/#sec-domhighrestimestamp)
+[**Monotonic Clock**](https://www.w3.org/TR/hr-time-2/#sec-monotonic-clock)
+Ilya Grigorik, James Simonsen, Jatinder Mann.
+[High Resolution Time Level 2](https://www.w3.org/TR/hr-time-2/) March 2018. W3C. URL: [https://www.w3.org/TR/hr-time-2/](https://www.w3.org/TR/hr-time-2/)
```
Comment (leobalter) [inline INTERPRETING.md]: this list is getting inline when rendered. can you fix it, please?

File: INTERPRETING.md [hunk 2]
```diff
@@ -93,6 +93,7 @@ properties of the global scope prior to test execution.
         sleeps the agent for approximately that duration.
       - **`leaving`** - a function that signals that the agent is done and
         may be terminated (if possible).
+      - **`monotonicNow`** - a function that returns a value that conforms to `DOMHighResTimeStamp` and is produced in such a way that its semantics conform to **Monotonic Clock**.
```
Comment (leobalter) [inline INTERPRETING.md]: From https://www.w3.org/TR/hr-time-2/#sec-domhighrestimestamp

> A DOMHighResTimeStamp SHOULD represent a time in milliseconds accurate enough to allow measurement while preventing timing attack

Should we ask for an even more precise clock when it's available?

Comment (leobalter) [reply INTERPRETING.md]: maybe link to this part: https://www.w3.org/TR/hr-time-2/#x7-monotonic-clock

Comment (rwaldron) [reply INTERPRETING.md]: Done

**Pull request merged**